canonical/ubuntu.com
# Copy update on /openstack/why-openstack

## Done

- Copy update on /openstack/why-openstack
- Break open source release graph out into an include module 

## QA

- Go to https://ubuntu-com-12342.demos.haus/openstack/why-openstack and compare page against [copy doc](https://docs.google.com/document/d/1StqylVToo1URkhLvNfl17vZ52H35fXCVOR1aMKG1aGU/edit#)
- Go to https://ubuntu-com-12342.demos.haus/about/release-cycle#openstack-release-cycle and check the chart displays properly

## Issue / Card

Fixes #https://github.com/canonical/commercial-squad/issues/752
